### PR TITLE
Remove display names from Profile model

### PR DESCRIPTION
**Pull request by Xyene**
Addresses #726.

WIP

### CODE REVIEW

File: judge/models/profile.py [hunk 1]
```diff
@@ -125,16 +124,10 @@ def calculate_points(self, table=(lambda x: [pow(x, i) for i in xrange(100)])(ge
 
     @cached_property
     def display_name(self):
-        if self.name:
-            return self.name
         return self.user.username
 
     @cached_property
     def long_display_name(self):
```
Comment (quantum5) [inline judge/models/profile.py]: Hopefully we can get rid of this too.

File: judge/models/profile.py [hunk 2]
```diff
@@ -125,16 +124,10 @@ def calculate_points(self, table=(lambda x: [pow(x, i) for i in xrange(100)])(ge
 
     @cached_property
     def display_name(self):
```
Comment (quantum5) [inline judge/models/profile.py]: Hopefully we can get rid of this too.

File: judge/models/profile.py (hunk 1 above)
Comment (Xyene) [reply judge/models/profile.py]: > WIP

Comment (quantum5) [reply judge/models/profile.py]: Of course, but this should be part of the work that is currently in progress.

**Pull request merged**